api4/team_test.go

```diff
@@ -386,6 +386,31 @@ func TestGetTeamStats(t *testing.T) {
 	CheckUnauthorizedStatus(t, resp)
 }
 
+func TestGetTeamUnread(t *testing.T) {
+	th := Setup().InitBasic().InitSystemAdmin()
+	defer TearDown()
+	Client := th.Client
+
+	user := th.BasicUser
+	Client.Login(user.Email, user.Password)
+
+	_, resp := Client.GetTeamUnread(model.NewRandomString(16))
+	CheckBadRequestStatus(t, resp)
```

Comment:
No that's the success case for a system admin. We need to make sure the route works for a regular user as well